templates/components/workflows/typescript/deep_research/workflow.ts

```
@@ -0,0 +1,445 @@
+
+# IMPORTANT when writing in markdown format:
++ Use tables or figures where appropriate to enhance presentation.
++ Preserve all citation syntax (the \`[citation:id]()\` parts in the provided context). Keep these citations in the final report - no separate reference section is needed.
++ Do not add links, a table of contents, or a references section to the report.
+`;
+
+// workflow events
+type ResearchQuestion = { questionId: string; question: string };
+type ResearchResult = ResearchQuestion & { answer: string };
+
+class PlanResearchEvent extends WorkflowEvent<{}> {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)